.github/workflows/build-graphscope-wheels-linux.yml

```diff
@@ -34,9 +50,9 @@ jobs:
 
         cd k8s
         # graphscope
-        sudo make graphscope-manylinux2014-py2-py3
+        sudo make graphscope-py3-package
```

Comment:
remove sudo should be OK, since it is `make` not make install.

Reply:
`sudo` is needed since `make install` exists in`graphscope-py3-package` target